src/online-members/script.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+let members = [];
+let memberTasks = [];
+let isTaskDataBeingFetched = false;
+
+function createTextNode(text) {
+  return document.createTextNode(text);
+}
+
+function createProfileImage(url, alt) {
+  const img = document.createElement('img');
+  img.classList = PROFILE_IMAGE_CLASS;
+  img.src = member.picture
+    ? getCloudinaryImgURL(url, RDS_PROFILE_IMAGE_SIZE)
+    : RDS_PROFILE_DEFAULT_IMAGE;
+  img.setAttribute('alt', alt);
+}
+
+function createMemberNode(member) {
+  const div = document.createElement('div');
+  div.classList = MEMBERS_CLASS_LIST;
+  div.dataset.username = member.username;
+  // div.appendChild(img);
+  div.appendChild(createTextNode(member.username));
```

Comment:
Nit: can we move `createTextNode(member.username` above and store it in a variable and then use it here

Reply:
Have done this in refactor PR  [here](https://github.com/Real-Dev-Squad/website-dashboard/pull/208/files#diff-0ad130b640282bfdf6bfd08eac4aa483083c7670083b149de30e0bc9fe3ba2ccR36-R50)